libpod/networking_linux.go

```diff
@@ -888,6 +888,8 @@ func (c *Container) getContainerNetworkInfo() (*define.InspectNetworkSettings, e
 		if err != nil {
 			return nil, err
 		}
+		netNsCtr.lock.Lock()
```

Comment:
Can you drop a comment above that mentions the issue number?  I find such breadcrumbs extremely helpful and time saving.

Reply:
done